app/views/artists/index.html.haml

```diff
@@ -1,16 +1,14 @@
-%h1 Listing artists
-
-%table
-  %tr
-    %th
-    %th
-    %th
-
-  - @artists.each do |artist|
-    %tr
-      %a{href: artist_path(artist)}
-        %h3= artist.name
-        = image_tag artist.profile_picture.url(:medium)
-
-%br
-
+%ul.grid-3-col
+	- @artists.each do |artist|
+		%li.mix.artist{data: {position: artist.id}}
+			- path = artist_path(artist)
+			= link_to path do
+				.tile
+					= image_tag artist.profile_picture.url(:tile), class: 'glitch'
+					.info
+						= artist.bookings
+						%h3.artist-name= artist.name
+						.location
+							//= schedule_for(artist)
```

Comment:
c'est noté.

*////////////////////////////////////////////////////////////////////////////////////////////////*

*|||||| architect EPF & parametric design consultancy||||||  mobile:
+41763652683     ||||||  www:  http://www.proceduralstrategies.net
<http://www.proceduralstrategies.net/> ||||||  mail:
hvanboetzelaer@gmail.com
<hvanboetzelaer@gmail.com>////////////////////////////////////////////////////////////////////////////////////////////////*

On Wed, Mar 4, 2015 at 11:11 AM, Walid <notifications@github.com> wrote:

> In app/views/artists/index.html.haml
> <https://github.com/walidvb/festor/pull/4#discussion_r25763780>:
>
> > -        = image_tag artist.profile_picture.url(:medium)
> > -
> > -%br
> > -
> > +%ul.grid-3-col
> > +	- @artists.each do |artist|
> > +		%li.mix.artist{data: {position: artist.id}}
> > +			- path = artist_path(artist)
> > +			= link_to path do
> > +				.tile
> > +					= image_tag artist.profile_picture.url(:tile), class: 'glitch'
> > +					.info
> > +						= artist.bookings
> > +						%h3.artist-name= artist.name
> > +						.location
> > +							//= schedule_for(artist)
>
> comments in haml: -# ou =#
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/walidvb/festor/pull/4/files#r25763780>.
>